packages/components/src/public/QueryModel/ViewMenu.tsx

```diff
@@ -4,16 +4,18 @@ import { DropdownButton, MenuItem } from 'react-bootstrap';
 import { QueryModel, ViewInfo } from '../..';
 import { blurActiveElement } from '../../internal/util/utils';
 import { getQueryMetadata } from '../../internal/global';
+import { isCustomizeViewsInAppEnabled } from '../../internal/app/utils';
 
 interface ViewMenuProps {
     hideEmptyViewMenu: boolean;
     model: QueryModel;
     onViewSelect: (viewName: string) => void;
+    onSaveView?: () => void;
```

Comment:
is this an optional prop? I think there was just the one usages of this component in GridPanel, so this could be required.